app/services/db/letter_requests_to_usps_ftp_log/letters_sent_in_range.rb

```diff
@@ -0,0 +1,9 @@
+module Db
+  module LetterRequestsToUspsFtpLog
+    class LettersSentInRange
+      def self.call(start_date, end_date)
+        ::LetterRequestsToUspsFtpLog.where(ftp_at: start_date..end_date)
```

Comment:
It looks like this class and method is only called in one place. What do you think of removing the class and using `::LetterRequestsToUspsFtpLog.where(ftp_at: start_date..end_date)` directly instead?